config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -635,7 +635,7 @@ periodics:
       - --scenario=kubernetes_e2e
       - --
       - --check-leaked-resources
-      - --env=KUBE_FEATURE_GATES=AllAlpha=true,DisableCloudProviders=false,DisableKubeletCloudCredentialProviders=false,InTreePluginGCEUnregister=false
+      - --env=KUBE_FEATURE_GATES=AllAlpha=true,RecoverVolumeExpansionFailure=false,DisableCloudProviders=false,DisableKubeletCloudCredentialProviders=false,InTreePluginGCEUnregister=false
```

Comment:
I'm not sure we need to set this, skipping the tests for it should be sufficient, right?

Reply:
Okay removed.  Just FYI, the reason for dropping the feature-gate was, once  we had a bug where if feature-gate is enabled in k8s (including kubelet) but external-resizer is older, then kubelet won't start resizing a volume until `nodeResizePending` is true. But then we fixed it, so we should be good.